src/operator/elemwise_op_common.h

```diff
@@ -156,6 +157,61 @@ inline bool ElemwiseStorageType(const nnvm::NodeAttrs& attrs,
     attrs, in_attrs, out_attrs);
 }
 
+template<int n_out>
```

Comment:
The size of out_attrs is determined when an op is registered by calling set_num_outputs. To me, this `n_out` is redundant information.